### ISSUE TITLE
Correct the loss function

### ISSUE THREAD

**Issue by radiachkik** [radiachk/MuZero#16]
For board games we omit the loss of the reward (because directly bootstrap into the final reward (same as value function)) and take the squarred error for the value function. This represents the error between the predicted and real end of the game.

For general MDP's (f.e. Atari),  we use the the cross entropy for the categorical representation of the reward and value.

For the policy loss we should also use the cross entropy.
